api/rda.py

```diff
@@ -71,6 +72,30 @@ def wrapper(body, **kwargs):
     return wrapper
 
 
+def endpoints(plugin=None):
```

Comment:
please move this function to the end of the file, so that the other methods that deal with the /rda* paths appear first

Reply:
Moved